drafts/use-cases/5.1.creating-new-event-indirectly.md

```diff
@@ -0,0 +1,128 @@
+## Creating new event indirectly
+
+### Story
+
+As an API publisher
+I want allow consumers to construct event's identifier
+So that they can control the .
+
+As an API consumer
+I want to be able to create new resources of a given type
+So I can produce content.
+
+### Usage
+
+``` js
+var event = {
+    "@context": "/api/context.jsonld",
+    "@type": "schema:Event",
+    "eventName": "My brand new event",
+    "eventDescription": "Hope it will work",
+    "startDate": "2017-04-19",
+    "endDate": "2017-04-19"
+};
+var templateVariables = {
+    "slug": [
+        "meeting",
+        "with-will"
+    ]
+};
+
+var client = new HydraClient();
+var operation = client.get("http://example.com/events")
+    .getOperationOfType('http://schema.org/CreateAction', /* recursive */ true)
+    .filter(operation => operation.expects === 'http://schema.org/Event');
+    
+client.invoke(operation, event, templateVariables);
+```
+
+### Details
+
+To create an `Event` using `PUT`, the collection resource has to be linked to the actual target of the operation.
+This linked object can be a concrete resource or an `IriTemplate` as presented in the above snippet. In this case 
+the representation of the collection resource is not unlike that seen in the [Searching events use case](7.searching-events.md)
+
+```http
+GET /api/events
+```
+
+```http
+HTTP 200 OK
+```
+
+``` json
+{
+    "@context": "/api/context.jsonld",
+    "@id": "/api/events",
+    "@type": "hydra:Collection",
+    "manages": {
+      "property": "rdf:type",
+      "object": "schema:Event"
+    },
+    "totalItems": 0,
+    "members": [ ],
+    "http://example.com/vocab/createEvent": {
+        "@type": "IriTemplate",
+        "hydra:template": "http://example.com/api/event{/slug*}",
+        "hydra:variableRepresentation": "hydra:BasicRepresentation",
+        "hydra:mapping": [
+            {
+                "@type": "hydra:IriTemplateMapping",
+                "variable": "slug",
+                "property": "xsd:String",
+                "required": true
+            }
+        ],
+        "hydra:operation": [
+          {
+            "@type": [ "hydra:Operation", "schema:CreateAction" ],
```

Comment:
Does it create a resource only? Or is it added indeed to the collection? It is exposed within the collection context, thus I'd assume that it would both create and add to the collection. Otherwise this operation could be also available in the supported classes context in which it would only mean that the new resource is going to be created.
From the other hand, how a pure `add` (that just adds a resource that already exists) operation would look like?